libs/sdk-backend-tiger/scripts/run-integrated-with-workspace.sh

```diff
@@ -0,0 +1,20 @@
+set -e
+
+if [[ -z "${WORKSPACE_ID}" ]]; then
+  npm run create-ref-workspaces
+  export $(xargs < .env)
+  WORKSPACE_CREATED=true
+else
+  echo "WORKSPACE_ID already set, skip creating new one!"
+fi
+
+# export variables for ci/cd
+export NO_COLOR=1
+
+# run the tests
+npm run integrated-test-live-backend
```

Comment:
Please copy the whole logic from `libs/sdk-backend-tiger/scripts/integrated-test-live-backend.sh` to here.
We don't want to call the test through many level of scripts